src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static void generateTaskStatus(String taskIcon, String statusIcon, String description) {
+        System.out.println("\t[" + taskIcon + "]" + "[" + statusIcon + "] " + description);
+    }
+    public static void drawLine() {
+        System.out.println("------------------------------------");
+    }
+    public static void addedMsg() {
+        System.out.println("Got it. I've added this task:");
+    }
+
+    public static void taskCountReminder(int noOfTasks) {
+        System.out.printf("Now you have %d tasks in the list.\n",noOfTasks);
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        drawLine();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        drawLine();
+        Scanner in = new Scanner(System.in);
+        String input;
+        Task[] taskArray = new Task[100];
+        int addCount = 0;
+        do {
```

Comment:
Consider abstracting the commonalities of the various conditional clauses. Maybe implement a switch case? :D